Spline 0.5.5 integration failing in `aws-poc`

## Describe the bug
Having built latest `aws-poc` 5da8e365 (where spline 0.5.5 was integrated) fails when running on EMR with the following exception:
```
20/12/14 10:55:01 INFO QueryExecutionEventHandlerFactory: Spline version: 0.5.5
Exception in thread "main" java.lang.NoClassDefFoundError: za/co/absa/commons/config/ConfigurationImplicits$ConfigurationRequiredWrapper$
        at za.co.absa.spline.harvester.conf.DefaultSplineConfigurer.splineMode$lzycompute(DefaultSplineConfigurer.scala:80)
        at za.co.absa.spline.harvester.conf.DefaultSplineConfigurer.splineMode(DefaultSplineConfigurer.scala:80)
        at za.co.absa.spline.harvester.QueryExecutionEventHandlerFactory.createEventHandler(QueryExecutionEventHandlerFactory.scala:35)
        at za.co.absa.spline.harvester.SparkLineageInitializer$SparkSessionWrapper.enableLineageTracking(SparkLineageInitializer.scala:49)
        at za.co.absa.enceladus.common.CommonJobExecution$class.prepareJob(CommonJobExecution.scala:98)
        at za.co.absa.enceladus.standardization.StandardizationJob$.prepareJob(StandardizationJob.scala:25)
        at za.co.absa.enceladus.standardization.StandardizationJob$.main(StandardizationJob.scala:38)
        at za.co.absa.enceladus.standardization.StandardizationJob.main(StandardizationJob.scala)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
        at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:853)
        at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:161)
        at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:184)
        at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:86)
        at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:928)
        at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:937)
        at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
Caused by: java.lang.ClassNotFoundException: za.co.absa.commons.config.ConfigurationImplicits$ConfigurationRequiredWrapper$
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 20 more
```


## To Reproduce
Steps to reproduce the behavior OR commands run:
1. Build & package `aws-poc` 5da8e365
2. Upload jar to EMR (currently xx.yy.zz.79)
3. Run a valid standardization job (spline setting (enabled, disabled) do not seem to matter)

## Expected behaviour
Behavior of version 0.5.4 - running normally.

## Additional context
Having tried reverting the `<spline.agent.version>` property in parent `pom.xml` to previous version 0.5.4 fixes the issue.

--- a/spark-jobs/pom.xml
+++ b/spark-jobs/pom.xml
@@ -26,7 +26,7 @@
     <properties>
         <scalastyle.configLocation>${project.parent.basedir}/scalastyle-config.xml</scalastyle.configLocation>
         <hyperdrive.version>3.0.0</hyperdrive.version>
-        <commons.version>0.0.10</commons.version>
+        <commons.version>0.0.15</commons.version>
         <spark.sql.kafka.version>2.4.5</spark.sql.kafka.version>
     </properties>
 

